### PR TITLE
[CM] Rename blacklist to non-offensive form

### LINKED ISSUE #10975: [CM] Rename "blacklist" for `reject_list`
Using blacklist is an offensive term and we should not use it, instead we need to rename it to `reject_list` or something similar.

### PR DESCRIPTION
**Pull request by michalpristas**
This PR renames blacklist in CM and substitutes with Rejectlist 

Fixes #10975

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -10,6 +10,33 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
...
+- ILM will be available by default if Elasticsearch > 7.0 is used. {pull}10347[10347]
+- Allow Central Management to send events back to kibana. {issue}9382[9382]
+- Initialize the Paths before the keystore and save the keystore into `data/{beatname}.keystore`. {pull}10706[10706]
+- Add `cleanup_timeout` option to docker autodiscover, to wait some time before removing configurations after a container is stopped. {issue]10374[10374] {pull}10905[10905]
+- On Google Cloud Engine (GCE) the add_cloud_metadata will now trim the project
+  info from the cloud.machine.type and cloud.availability_zone. {issue}10968[10968]
+- Empty `meta.json` file will be treated as a missing meta file. {issue}8558[8558]
+- Rename `migration.enabled` config to `migration.6_to_7.enabled`. {pull}11284[11284]
+- Beats Xpack now checks for Basic license on connect. {pull}11296[11296]
+- Central Management: Rename blacklist to non-offensive form {pull}11523[11523]
```
Comment (ycombinator) [inline CHANGELOG.next.asciidoc]: Rebase gone bad? :)

File: x-pack/libbeat/management/manager.go [hunk 2]
```diff
@@ -33,36 +33,42 @@ func init() {
...
+	mux        sync.RWMutex
 }
 
 // NewConfigManager returns a X-Pack Beats Central Management manager
 func NewConfigManager(config *common.Config, registry *reload.Registry, beatUUID uuid.UUID) (management.ConfigManager, error) {
 	c := defaultConfig()
+
 	if config.Enabled() {
+		if config.HasField("blacklist") {
+			logp.Warn("Blacklist configuration section is obsolete. Please use 'Rejectlist' instead.")
```
Comment (ycombinator) [inline x-pack/libbeat/management/manager.go]: Would it make sense to use `cfgwarn.Deprecate` here?

**Comment by ph** [comment]
@ycombinator yes, but I think I will just close that PR we already marked the feature as deprecated.

**Comment by ycombinator** [comment]
@ph In that case would you mind looking at the In Progress PRs on the Beats Kanban board? There are other CM PRs too that might need closing? That's where I found this one.

**Comment by ph** [comment]
@ycombinator I've cleaned them all.

**Pull request closed without merging**